### ISSUE TITLE
Problem in training

### ISSUE THREAD

**Issue by BeepBump** [HyeongseokSon1/SRFeat#11]
Hi, when I run main_gan_init, i got this error.
[](url)
![image](https://user-images.githubusercontent.com/39153839/57918788-0d194e80-78ca-11e9-8e02-03634a409ccf.png)
Can u help me fix this error?

**Comment by HyeongseokSon1** [comment]
It seems that loading validation imgs fails.
As shown at line 60 and 61 in main_gan_init.py, the default code use bmp imgs as validation.
You can change the file extension appropriate for your validation set.

**Comment by BeepBump** [comment]
Thanks!
I have solved the problem.
However, when I train the network, I add "tl.vis.save_images" to have a quick view.
It appears: Lossy conversion from float32 to uint8.
How can I solve this warning?

**Comment by HyeongseokSon1** [comment]
I think that no information will be lost, so it is no matter.
Because all input and ground-truth (label) images are 8bit PNG images, the network does produce as much information as 8bit even though they are converted to float32 tensors.
Anyway, we should save the result in 8bit PNG format, and there must be such conversion.

**Comment by xiaoyi-st** [comment]
> 似乎加载验证imgs失败。
> 如main_gan_init.py中的第60和61行所示，默认代码使用bmp imgs作为验证。
> 您可以更改适合您的验证集的文件扩展名。

Do I have to change the file extension of the verification set to bmp?
